### PR TITLE
Updated Texts and added sidebase documentation

### PR DESCRIPTION
**Pull request by zoey-kaiser**
This PR adds the sidebase documentation pages. The source of these is the current sidebase ReadMe. This PR also updates the Features to be more inline with what sidebase currently has to offer. It also adds a extra banner on the landing page to advertise any sidebase or SIDESTREAM related news.

TOOD:
Update testimonials.

### CODE REVIEW

File: components/content/Logo.vue [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+    <div class="self-center whitespace-nowrap leading-[10px] ml-2 dark:sidebase-green-400">
+      <h1 class="font-CooperHewitt my-0 text-2xl font-light leading-5">
+        sidebase
+      </h1>
+      <h2 class="my-0 pl-[1px] text-xs font-light">
+        by SIDESTREAM
```
Comment (BracketJohn) [inline components/content/Logo.vue:10]: where does this show up?

I cannot see it on the main page when I run it locally

**Review by BracketJohn — CHANGES_REQUESTED** [review]
1. I think the `sidebase` docs are still missing? could look like:

```
docs | modules | social-icons
```
clicking on docs: -> sidebase docs
clicking on modules -> sub-header with the three options `nuxt-auth`, `nuxt-session` and `nuxt-parse`
clicking on social icons -> whatever happens then

2. when I load this on local dev, I see the following page:
image

3. let's not say "the old documentation", rathjer: These docs will be migrated soon, YOU CAN FIND THE CURRENT DOCUMENTATION HERE(LINK)

**Comment by zoey-kaiser** [comment]
> I think the `sidebase` docs are still missing? could look like:

Debugged together in call. Removing the `.nuxt` folder fixed all the issues!

**Review by BracketJohn — COMMENTED** [review]
- please try to fix the length of [1]
- let's say: "for fullstack app development" in the tag line
- maybe we should already state a big "sidebase" earlier, like the tanstack people did? I ahve the feeling that at the moment its unclear what the tag line is talking about [2]
- features
    - "Feature rich Node backend" is not really a feature - can we make something more about prisma


[1]
image

[2] tanstack page / our page

image


image

**Comment by zoey-kaiser** [comment]
Screenshot 2022-11-29 at 15 32 14

File: content/1.sidebase/1.welcome/2.features.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- **Fullstack**: Develop frontend and backend in a single TypeScript code base
+- **Fast to code**: Database, example tests, example components and example pages are all there for you to fill out
+- **Fewer bugs**: Strong data-validation using `zod` to validate all transferred data, fully typed API-routes and strict DB models
+- **Easy to use**: Designed to follow best practices and to be ready-to-go for development, without additional dev-dependencies like `docker` that make it hard to get started
+- **Ready for launch**: Github Actions CI, Dockerfile, easy switch to most popular SQL-databases are all there, out of the box ([get in touch if you're missing something](https://github.com/sidebase/sidebase/issues/new/choose))
+::
+
+::list{type="success"}
+- Fullstack [`Vue 3`](https://vuejs.org/) + [`Nuxt 3 (stable)`](https://v3.nuxtjs.org/),
+- Database models, migrations, queries and easy DB-switching via [`TypeORM`](https://typeorm.io/),
```
Comment (BracketJohn) [inline content/1.sidebase/1.welcome/2.features.md:13]: Prisma, actually in v3 :D

**Comment by zoey-kaiser** [comment]
> please try to fix the length of [1]

Already attempted earlier today! Was not possible without making a new component. As discussed in text, ill do this in another step once we have V1 ready!

**Review by BracketJohn — COMMENTED** [review]
Im not going to review the docs now but reather will rework them later when v3-sidebase itself is done.

**Comment by zoey-kaiser** [comment]
> sidebase nuxt-auth nuxt-session nuxt-parse | blog | social-icons

Completely customizing the header is not as easy as it may seem (one of the downsides of using docus). Therefore I can change the order but cannot add any lines. The blog will be added with the PR #15 

> please try to fix the length of [1]

Already attempted earlier today! Was not possible without making a new component. As discussed in text, ill do this in another step once we have V1 ready!

>  can we make it clearer that this is copyable? eg via a copy icon (I think this is also a feature of docus)

Hovering over it revels a "copy" text. I am not sure what icon you are referring to, but I cannot modify the component.

**Comment by BracketJohn** [comment]
> Already attempted earlier today! Was not possible without making a new component. As discussed in text, ill do this in another step once we have V1 ready!

and have you also checked whether we can remove the `@latest` from the command (as also suggested earlier today) to save some room? (:

**Comment by zoey-kaiser** [comment]
> and have you also checked whether we can remove the `@latest` from the command (as also suggested earlier today) to save some room? (: If you attempt this please ensure that the command still works

Just checked this! The command works without. I changed it and now it fits into one line 🥳

File: content/1.sidebase/1.welcome/2.features.md (hunk 2 above)
Comment (zoey-kaiser) [reply content/1.sidebase/1.welcome/2.features.md]: Updated it in all locations!

**Review by BracketJohn — APPROVED** [review]

**Pull request merged**